File: baekjoon/baekJoon_3009.java
package baekjoon;

import java.io.*;
import java.util.StringTokenizer;

public class Main{
    static int [][] arr = new int [3][2];

    public static int solve(int target){
        if(arr[0][target] == arr[1][target]){
            return arr[2][target];
        }else if(arr[0][target] == arr[2][target]){
            return arr[1][target];
        }else{
            return arr[0][target];
        }
    }
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        for(int i = 0 ; i < 3 ; i++){
            StringTokenizer st = new StringTokenizer(br.readLine() , " ");
            for(int j = 0 ; j < 2 ; j++){
                arr[i][j] = Integer.parseInt(st.nextToken());
            }
        }
        bw.write(solve(0)+ " ");
        bw.write(solve(1) + "");
        bw.close();
    }
}

File: baekjoon/baekJoon_3009.kt
package baekjoon

import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.*

fun main(){
    val br = BufferedReader(InputStreamReader(System.`in`))

    var a=Array<Array<Int>>(3){Array<Int>(2){ _ ->0}}
    for(i in 0..2){
        val st = StringTokenizer(br.readLine())
        for(j in 0..1){
            a[i][j] = st.nextToken().toInt()
        }
    }
    var result = Array<Int>(2) { 0 }
    if(a[0][1] == a[1][1]){
        result[1] = a[2][1]
    }else if(a[2][1] == a[1][1]){
        result[1] = a[0][1]
    }else{
        result[1] = a[1][1]
    }
    if(a[0][0] == a[1][0]){
        result[0] = a[2][0]
    }else if(a[2][0] == a[1][0]){
        result[0] = a[0][0]
    }else{
        result[0] = a[1][0]
    }

    var y:Int = result[1]
    var x:Int = result[0]
    println("$x $y")

}
